### PR TITLE
feat(ScrollToTopButton): Move to _app.tsx

### PR DESCRIPTION
**Pull request by Woody810**
IMO, ScrollToTopButton should be a common button for all pages

### CODE REVIEW

File: packages/uikit/src/widgets/Menu/Menu.tsx [hunk 1]
```diff
@@ -206,6 +208,7 @@ const Menu: React.FC<React.PropsWithChildren<NavProps>> = ({
       <AtomBox display={{ xs: "block", md: "none" }}>
         <BottomNav items={links} activeItem={activeItem} activeSubItem={activeSubItem} />
       </AtomBox>
+      {createPortal(<ScrollToTopButtonV2 />, document.body)}
```
Comment (chefilip) [inline packages/uikit/src/widgets/Menu/Menu.tsx:211]: could you add a page filter? We might need to hide the buttongs in certain pages if needed?

Comment (Woody810) [reply packages/uikit/src/widgets/Menu/Menu.tsx]: I added isShowScrollToTopButton property, is that right?

**Comment by chefilip** [comment]
Hi @Woody810 

There is a failed build on Aptos site. Could you help me update it and then it's good to be merged? Thanks for your contribution.

**Comment by Woody810** [comment]
> Hi @Woody810
> 
> There is a failed build on Aptos site. Could you help me update it and then it's good to be merged? Thanks for your contribution.

done

**Comment by Woody810** [comment]
> Imo it would be better to add this component not in menu but in _app.tsx and we can enable/disable that in pages like the chainId filtering per page
> 
> 
> 
> https://github.com/pancakeswap/pancake-frontend/blob/ae6ed2811eaf564bd707f9f649eca186de8fee74/apps/web/src/pages/_app.tsx#L152
> 
> 

Let me check🙏

**Comment by Woody810** [comment]
> Imo it would be better to add this component not in menu but in _app.tsx and we can enable/disable that in pages like the chainId filtering per page
> 
> https://github.com/pancakeswap/pancake-frontend/blob/ae6ed2811eaf564bd707f9f649eca186de8fee74/apps/web/src/pages/_app.tsx#L152

done

File: packages/uikit/src/widgets/Menu/Menu.tsx [hunk 2]
```diff
@@ -84,6 +84,7 @@ const Menu: React.FC<React.PropsWithChildren<NavProps>> = ({
   activeSubItem,
   langs,
   buyCakeLabel,
+  isShowScrollToTopButton = true,
```
[the remark this answers is not in the document]
Comment (Woody810) [reply packages/uikit/src/widgets/Menu/Menu.tsx]: done

File: packages/uikit/src/widgets/Menu/types.ts [hunk 3]
```diff
@@ -29,6 +29,7 @@ export interface NavProps {
   cakePriceUsd?: number;
   currentLang: string;
   buyCakeLabel: string;
+  isShowScrollToTopButton?: boolean;
```
[the remark this answers is not in the document]
Comment (Woody810) [reply packages/uikit/src/widgets/Menu/types.ts]: done

**Review by chefilip — APPROVED** [review]

**Pull request merged**